program/check-upper-case/CheckUpperCase.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * 
+ * @param {string} character 
+ * @returns {boolean} whether `character` is uppercase or nott
+ */
+function checkUpperCaseInTS(character) {
```

Comment:
This is a JS file, but the function name included TS. Also, you could make it `checkUpperCase` as the function name instead.

Reply:
I did try this, but it gave me an error saying that the function name was already declared. I could do this by adding an `export {}` at the end of the file, if that is fine!

Reply:
It's the "cannot redeclare block-scoped variable" error, by the way.

Reply:
Fixed, by the way! There should be a lint error, still though.

Reply:
> It's the "cannot redeclare block-scoped variable" error, by the way

I think it should be ok since there are JS and TS files.  Thank you for editing!